opennms-base-assembly/src/main/filtered/etc/jmx-datacollection-config.d/jvm.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0"?>
+<jmx-datacollection-config rrdRepository="${install.share.dir}/rrd/snmp/">
+    <jmx-collection name="jvm">
```

Comment:
@mfuhrmann can you please double check here? The OpenNMS-JVM is looking for a `jsr160` collection and it seems like this is renamed here to `jvm` see [here](https://github.com/OpenNMS/opennms/blob/a0fd2dcbfeed754fd8b8a1188d8123fc74e3c569/opennms-base-assembly/src/main/filtered/etc/collectd-configuration.xml#L112)